cylc/flow/host_select.py

```diff
@@ -128,6 +128,13 @@ def select_workflow_host(cached=True):
     # be returned with the up-to-date configuration.
     global_config = glbl_cfg(cached=cached)
 
+    # condemned hosts may be suffixed with an "!" to activate "force mode"
+    blacklist = []
```

Comment:
prefer `condemn_list`?

Reply:
The name `blacklist` is used elsewhere in the file, so using the same name here for consistency. To explain:

The `select_host` method below is abstract (as we may have other uses for this beyond Cylc server selection) so uses the generic name `blacklist`.

This is the concrete method for Cylc server selection. It populates the `blacklist` argument of `select_host` from condemned hosts found in the configuration.